src/test/java/racingcar/domain/car/power/FixedPower.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain.car.power;
+
+import racingcar.domain.car.power.Power;
+
+public class FixedPower implements Power {
+    private boolean movable;
+
+    public FixedPower() {
+        this.movable = false;
+    }
+
+    @Override
+    public boolean isMovable() {
+        movable = !movable;
+        return movable;
```

Comment:
```suggestion
        return true;
```
항상 True를 반환하게 될것 같은데 movable의 값을 생성자를 통해 주입받는것도 좋아보여요